dnf-behave-tests/features/plugins-core/debug-dump.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Test for debug plugin - dumping
+
+
+Scenario: dnf debug-dump dumps file with configuration
+  Given I enable plugin "debug"
+    And I use repository "debug-plugin"
+    And I successfully execute dnf with args "install kernel-4.19.1 kernel-4.20.1"
+   When I execute dnf with args "debug-dump {context.dnf.tempdir}/dump.txt"
+   Then the exit code is 0
+    And file "//{context.dnf.tempdir}/dump.txt" matches line by line
+    """
+    dnf-debug-dump version 1
+    %%%%SYSTEM INFO
+      uname: .*
+      rpm ver: .*
+      python ver: .*
+    %%%%DNF INFO
+      arch: .*
+      basearch: .*
+      releasever: .*
+      dnf ver: .*
+      enabled plugins: debug
+      global excludes: 
+    %%%%RPMDB PROBLEMS
+    %%%%RPMDB
+      kernel-0:4\.19\.1-fc29\.x86_64
+      kernel-0:4\.20\.1-fc29\.x86_64
+    %%%%REPOS
+    %debug-plugin - file:///home/mblaha/work/debug-restore-bz1844533/ci-dnf-stack/dnf-behave-tests/fixtures/repos/debug-plugin
+      excludes: 
+      kernel-0:4.18.1-fc29.src
+      kernel-0:4.18.1-fc29.x86_64
+      kernel-0:4.19.1-fc29.src
+      kernel-0:4.19.1-fc29.x86_64
+      kernel-0:4.20.1-fc29.src
+      kernel-0:4.20.1-fc29.x86_64
+      test-install-0:1-fc29.src
+      test-install-0:1-fc29.x86_64
+      test-remove-0:1-fc29.src
+      test-remove-0:1-fc29.x86_64
+      test-replace-0:1-fc29.src
+      test-replace-0:1-fc29.x86_64
+      test-replace-0:2-fc29.src
+      test-replace-0:2-fc29.x86_64
+      test-replace-0:3-fc29.src
+      test-replace-0:3-fc29.x86_64
+    %%%%RPMDB VERSIONS
+      all: 2:9a5b819daf813e63ecfd470f07bcacb534b506d5
```

Comment:
This hash doesn't work when I tried to run it, I don't think we can rely on it being always the same, there is too many moving parts.